src/main/java/org/sagebionetworks/bridge/exporter/dynamo/DynamoHelper.java

```diff
@@ -0,0 +1,133 @@
+package org.sagebionetworks.bridge.exporter.dynamo;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import javax.annotation.Resource;
+
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.Table;
+import com.jcabi.aspects.Cacheable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import org.sagebionetworks.bridge.exporter.exceptions.SchemaNotFoundException;
+import org.sagebionetworks.bridge.exporter.metrics.Metrics;
+import org.sagebionetworks.bridge.json.DefaultObjectMapper;
+import org.sagebionetworks.bridge.schema.UploadSchema;
+import org.sagebionetworks.bridge.schema.UploadSchemaKey;
+
+/**
+ * Encapsulates common calls Bridge-EX makes to DDB. Some of these have a little bit of logic or require marshalling
+ * from a DDB item to a Bridge-EX object. Others are there just for convenience, so we have all or most of our DDB
+ * interactions through a single class.
+ */
+@Component
+public class DynamoHelper {
+    private static final Logger LOG = LoggerFactory.getLogger(DynamoHelper.class);
+
+    private Table ddbParticipantOptionsTable;
+    private Table ddbSchemaTable;
+    private Table ddbStudyTable;
+
+    /** Participant options table, used to get user sharing scope. */
+    @Resource(name = "ddbParticipantOptionsTable")
+    public final void setDdbParticipantOptionsTable(Table ddbParticipantOptionsTable) {
```

Comment:
Can these be made package private? It seems like they can, and it's nice not to see them in autocompletion.